templates/CRM/common/paymentBlock.tpl

```diff
@@ -85,9 +85,11 @@
       var $form = $('#billing-payment-block').closest('form');
       {/literal}
       {if $contributionPageID}
-        {capture assign='contributionPageID'}id={$contributionPageID}&{/capture}
+        {capture assign='pageID'}id={$contributionPageID}&module=CiviContribute&{/capture}
```

Comment:
I thought there must be some reason use of ```id``` although I didn't found any. So I kept it as it is for reusability so in near future with the help of this entityID we can fetch other info. for other purpose rather than just passing custom_pre_id. I know that simply passing profile id will need less code but what if we need to gather other values for $this->_values !!